src/Tokenizer/Analyzer/ClassAnalyzer.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tokenizer\Analyzer;
+
+use PhpCsFixer\Tokenizer\Analyzer\Analysis\TypeAnalysis;
+use PhpCsFixer\Tokenizer\Tokens;
+use PhpCsFixer\Tokenizer\TokensAnalyzer;
+
+/**
+ * @internal
+ */
+final class ClassAnalyzer
+{
+    /**
+     * @param \PhpCsFixer\Tokenizer\Tokens $tokens
+     * @param $classIndex
+     *
+     * @return array
+     */
+    public function getClassDefinition(Tokens $tokens, $classIndex)
+    {
+        return $this->getClassDefinitionType($tokens, $classIndex);
```

Comment:
no proxy methods, please.
this brings no value

Reply:
Fixed in #4362